.eslintrc.js

```diff
@@ -6,18 +6,18 @@ module.exports = {
   },
   plugins: [
     'ember',
-    'prettier',
+    // 'prettier',
   ],
   extends: [
     'eslint:recommended',
     'plugin:ember/recommended',
-    'prettier',
+    // 'prettier',
   ],
   env: {
     browser: true
   },
   rules: {
-    'prettier/prettier': 'error',
```

Comment:
I haven't figured this out yet...

Without it the build is failing with prettier running on `/addon`:

```
  - broccoliBuilderErrorStack: Error: ENOENT: no such file or directory, scandir '/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/addon/addon/helpers'
    at Object.readdirSync (fs.js:790:3)
    at traverseFolder (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32564:18)
    at findRoot (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32580:10)
    at maybeParse (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32592:16)
    at editorconfigSyncNoCache (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:32605:35)
    at editorconfigSyncNoCache (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:28538:20)
    at /Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34796:14
    at Array.map (<anonymous>)
    at _resolveConfig (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34795:40)
    at Function.resolveConfig.sync (/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/node_modules/prettier/index.js:34831:12)
  - codeFrame: ENOENT: no such file or directory, scandir '/Users/simonihmig/Projects/ember-angle-bracket-invocation-polyfill/addon/addon/helpers'
  - errorMessage: Build Canceled: Broccoli Builder ran into an error with `broccoli-persistent-filter:EslintValidationFilter` plugin. 💥
```

I already tried upgrading related dependencies, but without success so far. Any ideas? 🤔

Reply:
Remove `ember-cli-eslint`, add `eslint`.

Reply:
https://github.com/ember-cli/ember-cli-eslint/issues/235 for background (and https://github.com/emberjs/rfcs/blob/master/text/0121-remove-ember-cli-eslint.md).